db/migrate/20170613144708_add_sti_to_container_image.rb

```diff
@@ -0,0 +1,21 @@
+class AddStiToContainerImage < ActiveRecord::Migration[5.0]
+  class ContainerImage < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    say_with_time("Adding type and ems_ref columns to ContainerImage and setting type") do
```

Comment:
The say_with_time should only be around the data migration but...not the add_columns (otherwise you are hiding the logging of the add_column calls).